src/twisted/internet/cfreactor.py

```diff
@@ -459,7 +517,8 @@ def iterate(self, delay=0):
         by letting the loop run for a little while and then scheduling a timed
         call to exit it.
         """
-        self.callLater(delay, self._stopNow)
+        self._started = True
+        self.callLater(0, self.crash)
```

Comment:
is the `delay` ignored on purpose? 
... and why do we call crash on `itearate` ?

I think that this need more comments :)

Reply:
2be0fc7aca